lib/linguist/languages.yml

```diff
@@ -4056,6 +4078,7 @@ Scheme:
   color: "#1e4aec"
   extensions:
   - ".scm"
+  - ".sch"
```

Comment:
I trust she knows what she's doing. 👍 'Cause I don't, haha. Thanks!

(I named [one of the samples](https://github.com/github/linguist/pull/3743/files#diff-b5373d71f52525d8004154f132391210) after her, BTW... LOL)